ui/admin/app/routes/scopes/scope/sessions/index.js

```diff
@@ -4,5 +4,174 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 
-export default class ScopesScopeSessionsIndexRoute extends Route {}
+export default class ScopesScopeSessionsIndexRoute extends Route {
+  // =services
+
+  @service store;
+
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    users: {
+      refreshModel: true,
+      replace: true,
+    },
+    targets: {
+      refreshModel: true,
+      replace: true,
+    },
+    status: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  allSessions;
+  associatedUsers;
+  associatedTargets;
+
+  /**
+   * Loads all sessions under the current scope and encapsulates them into
+   * an array of objects containing their associated users and targets.
+   * @return {Promise{[{sessions: [SessionModel], allSessions: [SessionModel], associatedUsers: [UserModel], associatedTargets: [TargetModel], totalItems: number}]}}
+   */
+  async model({ search, users, targets, status, page, pageSize }) {
+    const { id: scope_id } = this.modelFor('scopes.scope');
+    const filters = {
```

Comment:
Shouldn't the scope id filter be added here?

Reply:
Yes they should. Thanks for catching that.